src/prefabs/tabs.js

```diff
@@ -13,6 +13,12 @@
           value: '1',
           type: 'NUMBER',
         },
+        {
```

Comment:
Same feedback as other pr, maybe this could go under advanced settings?

Reply:
Don't think it should be under advanced as its probably one of the most used options. 

But please rename it to "Selected design tab index" as we communicate working in the Page Builder officially as "Design mode"